InferenceSystem/README.md

```diff
@@ -68,13 +73,35 @@ export AZURE_COSMOSDB_PRIMARY_KEY="<yourprimarykey>"
 
 ```
 cd InferenceSystem
-python LiveInferenceOrchestrator.py
+python LiveInferenceOrchestrator.py --config ./config/Test/FastAI_LiveHLS_OrcasoundLab.yml
+```
+
+You should see the following logs in your terminal. Since this is a Test config, no audio is being uploaded to Azure and not metadata is being written to CosmosDB.
```

Comment:
```suggestion
You should see the following logs in your terminal. Since this is a Test config, no audio is being uploaded to Azure and no metadata is being written to CosmosDB.
```